client/app/visualizations/chart/chart-editor.html

```diff
@@ -263,3 +267,35 @@
       </tbody>
     </table>
   </div>
+
+  <div ng-show="currentTab == 'colors'">
+    <table class="table table-condensed col-table">
+      <thead>
+      <th>zIndex</th>
+      <th>Label</th>
+      <th>Color</th>
+      </thead>
+      <tbody ui-sortable ng-model="form.colorsList">
+      <tr ng-repeat="name in form.colorsList">
+        <td style="cursor: move;"><i class="fa fa-arrows-v"></i> <span
```

Comment:
I notice this project has inline styles sprinkled throughout but it's a really bad idea since overriding them later becomes quite tricky.